src/components/Pill/Pill.tsx

```diff
@@ -0,0 +1,35 @@
+import { getStatusColor } from "@dashboard/misc";
+import { makeStyles, Pill as MacawuiPill, PillProps } from "@saleor/macaw-ui";
+import { useTheme, vars } from "@saleor/macaw-ui/next";
+import clsx from "clsx";
+import React from "react";
+
+const useStyles = makeStyles<{
+  color: PillProps["color"];
+}>(
+  {
+    pill: {
+      borderRadius: "32px",
+      border: "none",
+      backgroundColor: ({ color }) => `${color} !important`,
+    },
+  },
+  { name: "Pill" },
+);
+
+// Main purpose of this component is to override default Pill component
+// from macaw-ui to add custom styles
+// TODO: migrate to Pill component from new macaw-ui when it will be ready
+export const Pill = ({ color, ...props }: PillProps) => {
+  const { theme: currentTheme } = useTheme();
+  const backgroundColor = getStatusColor(color, currentTheme);
+  const classes = useStyles({
+    color: backgroundColor.startsWith("#")
```

Comment:
The problem is that we use here some values hardcoded because we don't have them in macaw
```
const COLOR_WARNING = "#FBE5AC";
const COLOR_WARNING_DARK = "#3E2F0A";
```

Reply:
sure - but then `getStatusColor` can return either `hsla` or `hex` value and you don't need to add `startsWith` logic